tests/payments.py

```diff
@@ -58,7 +58,7 @@ def test_delete_payment_type(self):
 
         self.assertEqual(response.status_code, status.HTTP_204_NO_CONTENT)
 
-        url = "/paymenttypes"
+        url = "/paymenttypes/1500"
```

Comment:
Did you mean to change this?

Reply:
no I didnt. I changed it back and pushed it up

Reply:
amazing catch!